### ISSUE TITLE
ftpasswd fails to restore password file permissions in some cases

### ISSUE THREAD

**Issue by Castaglia** [proftpd/proftpd#898]
Jacek Pieniazek reported, via email, that using `ftpasswd` (from ProFTPD 1.3.6) to change the password of an existing user, to the _same_ value, results in the password file having 644 permissions, rather than having having the permissions restored to 440:
```
$ ftpasswd --file /etc/proftpd/ftpd.passwd --stdin --passwd --name user --change-password
```
Similarly, when deleting a user which does not exist:
```
$ ftpasswd --file /etc/proftpd/ftpd.passwd --passwd --name user --delete-user
```
the permissions of the password file are not properly restored.

**Comment by Castaglia** [comment]
Fixed in master, and backported to the 1.3.6 branch.

**Comment by lordspace** [comment]
I decided to copy the ftpasswd Perl script from Ubuntu 20 because there's no official update yet.
https://github.com/qsandbox/proftpd_ftpasswd

I am wondering why there's no update for Ubuntu 18 proftpd package yet?
